acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+# puppetserver seems to take about 45s to start up
+timout_length = "60s"
```

Comment:
It's possible that 60 seconds is not long enough depending on the hardware; on the other hand I'd hate to force the test to be longer than that unless we know it's necessary.

Can we assume that Puppet Server will have already been started at least once prior to this test running?  If so, we know that all of the SSL files should be generated before this test runs, which should hopefully prevent the startup time from being too crazy.